langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -18,6 +18,16 @@ import ballerina/jballerina.java;
 
 # Returns the sum of zero or more decimal values.
 #
+# ```
+# decimal:sum(1.2, 2.3, 3.4) ⇒ 6.9
+#
+# decimal[] scores = [11.1, 22.2, 33.3];
+# decimal:sum(...scores) ⇒ 66.6
```

Comment:
```suggestion
# decimal:sum(...[11.1, 22.2, 33.3]) ⇒ 66.6
```

Reply:
I think introducing a variable is more clear in here.
Otherwise it may be confusing to user. wdyt?